### PR TITLE
add content for new WinForms APIs

### PR DESCRIPTION
**Pull request by mairaw**
Related to https://github.com/dotnet/docs/issues/4876

### CODE REVIEW

File: xml/System.Windows.Forms/DataGridViewComboBoxCell.xml [hunk 1]
```diff
@@ -149,9 +149,16 @@
...
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a new <see cref="T:System.Windows.Forms.AccessibleObject" /> for this <see cref="T:System.Windows.Forms.DataGridViewComboBoxCell" /> instance.</summary>
+        <returns>A new <see cref="T:System.Windows.Forms.AccessibleObject" /> instance that supports the ControlType UIA property.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The new <xref:System.Windows.Forms.AccessibleObject> object is only available in applications that are recompiled to target versions of the .NET Framework starting with the .NET Framework 4.7.2. Applications that target an earlier version of the .NET Framework but are running on the .NET Framework 4.7.2 can opt in to this feature by using a compatibility switch. For additional information, see  [What's New in Accessibility in the .NET Framework](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewComboBoxCell.xml]: This should be rephrased -- it implies that AccessibleObject itself is new. It's better to say something like, "The \<xref:System.Windows.Forms.AccessibleObject> returned by the \`CreateAccessibility\` method is only available..."

File: xml/System.Windows.Forms/DataGridViewTextBoxCell.xml [hunk 2]
```diff
@@ -123,9 +123,16 @@
...
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a new <see cref="T:System.Windows.Forms.AccessibleObject" /> for this <see cref="T:System.Windows.Forms.DataGridViewTextBoxCell" /> instance.</summary>
+        <returns>A new <see cref="T:System.Windows.Forms.AccessibleObject" /> instance that supports the ControlType UIA property.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The new <xref:System.Windows.Forms.AccessibleObject> object is only available in applications that are recompiled to target versions of the .NET Framework starting with the .NET Framework 4.7.2. Applications that target an earlier version of the .NET Framework but are running on the .NET Framework 4.7.2 can opt in to this feature by using a compatibility switch. For additional information, see  [What's New in Accessibility in the .NET Framework](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewTextBoxCell.xml]: This has the same problem as earlier.

File: xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml [hunk 3]
```diff
@@ -120,9 +120,16 @@
...
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a new <see cref="T:System.Windows.Forms.AccessibleObject" /> for this <see cref="T:System.Windows.Forms.DataGridViewTextBoxEditingControl" /> instance.</summary>
+        <returns>A new <see cref="T:System.Windows.Forms.AccessibleObject" /> instance that supports the ControlType UIA property.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The new <xref:System.Windows.Forms.AccessibleObject> object is only available in applications that are recompiled to target versions of the .NET Framework starting with the .NET Framework 4.7.2. Applications that target an earlier version of the .NET Framework but are running on the .NET Framework 4.7.2 can opt in to this feature by using a compatibility switch. For additional information, see  [What's New in Accessibility in the .NET Framework](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml]: Again...

File: xml/System.Windows.Forms/DataGridViewComboBoxCell.xml (hunk 1 above)
Comment (Tanya-Solyanik) [reply xml/System.Windows.Forms/DataGridViewComboBoxCell.xml]: I agree with Ron, word `new` is ambiguous,  but I have 2 additional points: method name is `CreateAccessibilityInstance`, and this method had been always available but it was inherited from the base class. in this release, we override this method and when the app opts in into the accessibility improvements, we return a object specific to the DataGridViewComboBoxCell, as opposed to object that works with any DataGridViewCell. Perhaps rephrase it like this:  "The <xref:System.Windows.Forms.AccessibleObject> returned by the `CreateAccessibilityInstance` method supports the `ControlType` UIA property only in applications that..."

Comment (mairaw) [reply xml/System.Windows.Forms/DataGridViewComboBoxCell.xml]: UIA stands for User Interface Automation @Tanya-Solyanik? I'm a bit confused on "the CreateAccessibilityInstance method supports the ControlType UIA property". (Note that I'm not the original author of this, I'm helping export from MSDN)

Comment (Tanya-Solyanik) [reply xml/System.Windows.Forms/DataGridViewComboBoxCell.xml]: Yes, you are right. usually it's called UI Automation, here is the link:
https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-overview

**Comment by mairaw** [comment]
I've addressed the feedback but I'm not 100% confident I got it right. Returns value still feels wrong since there are certain conditions to be met for that to be true.

Also, what happens if your app is not targeting 4.7.2 or using a compat switch?

@rpetrusha @Tanya-Solyanik

File: xml/System.Windows.Forms/DataGridViewComboBoxCell.xml [hunk 4]
```diff
@@ -149,9 +149,19 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a new <see cref="T:System.Windows.Forms.AccessibleObject" /> for this <see cref="T:System.Windows.Forms.DataGridViewComboBoxCell" /> instance.</summary>
+        <returns>A <see cref="T:System.Windows.Forms.AccessibleObject" /> instance that supports the <see cref="T:System.Windows.Automation.ControlType" /> UI Automation property.</returns>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewComboBoxCell.xml]: nit: A --> An

File: xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml (hunk 3 above)
Comment (rpetrusha) [reply xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml]: Although this is a new object, I'd delete "new" from the Remarks.

**Comment by rpetrusha** [comment]
I think I misunderstood your question, @mairaw. If an app doesn't target a .NET Framework version starting with 4.7.2, the default is legacy behavior; for apps that target an earlier version, the switch is only meaningful if they're running on 4.7.2 and want to opt into new accessibility behaviors.

**Pull request merged**